roottest/root/io/filemerger/testSimpleFile.C

```diff
@@ -1,7 +1,11 @@
 #ifdef R__HAS_DEFAULT_LZ4
 constexpr int expectedcomplevel = 404;
 #elif defined R__HAS_DEFAULT_ZLIB
+#if defined R__HAS_CLOUDFLARE_ZLIB
 constexpr int expectedcomplevel = 101;
+#else
+constexpr int expectedcomplevel = 101;
+#endif
```

Comment:
What is the intent here? (it seems be a NFC ...)

Reply:
It was just to visibilize the architecture but i can remove it, it's unneeded